src/components/home/devRevenueCosts/Development.scss

```diff
@@ -0,0 +1,285 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap');
+@import '../../../utilis/colors';
+@import '../../../utilis/mixins';
+
+.container{
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+}
+
+.right-side{
+    flex-grow: 1;
+    padding-top: 20px;
+    padding-left: 20px;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-right: 20px;
+}
+
+.home-text{
+    padding-bottom: 30px;
+    @include gilroy-medium;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 40px;
+    letter-spacing: 0em;
+    text-align: left;
+
+}
+
+.selections{
+    display: flex;
+    flex-direction: row;
+    padding-bottom:18px;
+}
+
+.selection-options{
+    @include inter-medium600();
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: $light-selection-color;
+
+    display: flex;
+    flex-direction: row;
+    border: 1px solid $green-border-color;
+    border-radius: 6px;
+    cursor: pointer;
+}
+
+.selection1{
```

Comment:
you have almost same style for all 3 sections. can we use just one

Reply:
I just copy paste Home.scss, but I will correct it

Reply:
I copied Home.scss, but I will correct it